### PR TITLE
nova: add page

### PR DESCRIPTION
**Pull request by jay-tau**
Helps with https://github.com/tldr-pages/tldr/issues/3689

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/nova.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`nova list`
+
+- List VMs of all tenants (admin user only):
+
+`nova list --all-tenants`
+
+- Boot a VM on a specific host:
+
+`nova boot --nic net-id=<net_id> --image <image_id> --flavor <flavor> --availability-zone nova:<host_name> <vm_name>`
```
Comment (kbdharun) [inline pages/linux/nova.md:16]:
```suggestion
`nova boot --nic net-id={{net_id}} --image {{image_id}} --flavor {{flavor}} --availability-zone nova:{{host_name}} {{vm_name}}`
```

File: pages/linux/nova.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Stop a server:
+
+`nova stop <server>`
```
Comment (kbdharun) [inline pages/linux/nova.md:20]:
```suggestion
`nova stop {{server}}`
```

File: pages/linux/nova.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Start a server:
+
+`nova start <server>`
```
Comment (kbdharun) [inline pages/linux/nova.md:24]:
```suggestion
`nova start {{server}}`
```

File: pages/linux/nova.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Attach a network interface to a specific VM:
+
+`nova interface-attach --net-id <net_id> <server>`
```
Comment (kbdharun) [inline pages/linux/nova.md:28]:
```suggestion
`nova interface-attach --net-id {{net_id}} {{server}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, I have some suggestions for this page.

File: pages/linux/nova.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# nova
+
+> The OpenStack project that provides a way to provision compute instances (aka virtual servers).
```
Comment (kbdharun) [inline pages/linux/nova.md:3]:
```suggestion
> The OpenStack project that provides a way to provision compute instances.
```
I don't think mentioning `virtual servers` is necessary.

File: pages/linux/nova.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Boot a VM on a specific host:
+
+`nova boot --nic net-id={{net_id}} --image {{image_id}} --flavor {{flavor}} --availability-zone nova:{{host_name}} {{vm_name}}`
+
+- Stop a server:
+
+`nova stop {{server}}`
+
+- Start a server:
```
Comment (kbdharun) [inline pages/linux/nova.md:22]: I think the start command should be first then the stop command. Can you rearrange them?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**